src/image.tsx

```diff
@@ -151,7 +151,8 @@ export const ImageContent = React.forwardRef<HTMLImageElement, ImageProps>(
             ? '50%'
             : noBorderRadius
             ? '0px'
-            : fallbackVar(styles.vars.mediaBorderRadius, vars.borderRadii.container);
+            : fallbackStyles(styles.vars.mediaBorderRadius, vars.borderRadii.container);
```

Comment:
an alternative fix would be to move these fallbackVar calls to the image.css.ts and video.css.ts files instead

Reply:
I tried but this is not so simple. Some components expect the borderRadius to be passed via props and using a className would require a bigger refactor

Reply:
Ok, we can merge with this approach, I've created a ticket to review this:
https://jira.tid.es/browse/WEB-1679